spec/dummy_app/app/models/person.rb

```diff
@@ -49,5 +49,10 @@ def load(value)
     end
   end
 
-  serialize :time_zone, TimeZoneSerializer.new
+  # Rails 7.1 deprecates positional argument for coder
+  if Rails.version >= "7.1"
```

Comment:
This check should be changed as we cannot compare version strings this way. At least, not correctly.

```ruby
"7.2.1" >= "7.10.1"
=> true

# Suggestion
Gem::Version.new(Rails.version) >= Gem::Version.new('7.1')
```

Reply:
Agreed! Could btw also be written like so:
Rails.gem_version >= Gem::Version.new("7.1")